doc/source/ray-observability/ray-metrics.rst

```diff
@@ -87,6 +91,38 @@ You can then see the default dashboard by going to dashboards -> manage -> Ray -
 .. image:: images/graphs.png
     :align: center
 
+Using an existing Grafana instance
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+When you want to use existing Grafana instance, before starting your Ray cluster you will need to setup environment variable `RAY_GRAFANA_HOST` with an URL of your Grafana. After starting Ray, you can find Grafana dashboard json at `/tmp/ray/session_latest/metrics/grafana/dashboards/default_grafana_dashboard.json`. `Import<https://grafana.com/docs/grafana/latest/dashboards/manage-dashboards/#import-a-dashboard>`_ this json to your Grafana.
```

Comment:
``Import<https://grafana.com/docs/grafana/latest/dashboards/manage-dashboards/#import-a-dashboard>`_`

Can you tell me a bit about what this exactly means? the URL seems to be linked to the current documentation

Reply:
Json at `/tmp/ray/session_latest/metrics/grafana/dashboards/default_grafana_dashboard.json` is encoding Grafana dashboard. When using existing Grafana, user needs to manually import that json. It is pretty straightforward, but Grafana documentation is showing how it is done, step-by-step. I am going to rephrase it.